recipes/bids-validator/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "bids-validator" %}
+{% set version = "1.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bids-standard/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3230732bf7b80bcd9fe4b879cb34f80468d1181aae7a79545ae1759baa4a4ddd
+
+build:
+  noarch: generic
+  number: 0
+  script: "npm install -g ."
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Do you need a compiler? If you do, it probably can't be `noarch: generic`
```suggestion
```

Reply:
Early on some package was calling g++ locally but I removed the requirement and it worked fine.

Thank you for the feedback.